.fantomasignore

```diff
@@ -0,0 +1 @@
+TokenParser.fs
```

Comment:
Why is this required?

Reply:
I discovered https://github.com/fsprojects/fantomas/issues/1211 so that is why I needed to exclude this from the `fantomas --check` we do inside the build script. A necessary evil until the next version drops.

Reply:
After refactoring I do have the `elif` without `else` constructs anymore so I was able to remove the `.fantomasignore`.